server/model/model.js

```diff
@@ -55,9 +56,34 @@ const addListing = function(data) {
     interaction_guests_details: data.interaction,
     other_details: data.other,
     avatar: data.avatar
-  });
+  })
+    .catch((err) => console.log(err));
+};
+
+const addBulkListing = (data) => {
+  return Listing.bulkCreate(data)
+    .catch((err) => console.log(err));
+};
+
+const deleteListing = (id) => {
+  return Listing.destroy({
+    where: {id: id}
+  })
+    .catch((err) => console.log(err));
+};
+
+const updateListing = (data) => {
+  let {id, ...changes} = data;
```

Comment:
This is really cool!

Reply:
And randomly this came up on the podcast I listened to on my way home last night... https://syntax.fm/ Episode 43 is about 20 JS methods to make you a better developer, including spread operator with objects!